Project/demo/src/main/java/com/example/demo/controller/QuestionController.java

```diff
@@ -0,0 +1,36 @@
+package com.example.demo.controller;
+
+import com.example.demo.model.Question;
+import com.example.demo.service.QuestionService;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+@Controller
+public class QuestionController
+{
+    @Autowired
+    private QuestionService questionService;
+
+    @RequestMapping(value = "/saveQuestion", method = RequestMethod.POST)
+    @ResponseBody
+    public ResponseEntity<String> Questions(@RequestBody Question question)
```

Comment:
Minor: change the name of the method from Question to for ex: addQuestion as we discussed previuosly.